vars/stageStartOpenshiftBuild.groovy

```diff
@@ -7,8 +7,23 @@ def call(def context, def buildArgs = [:], def imageLabels = [:]) {
 
     timeout(context.openshiftBuildTimeout) {
       patchBuildConfig(context, buildArgs, imageLabels)
-      sh "oc start-build ${context.componentId} --from-dir docker --follow -n ${context.targetProject}"
+      sh (script: "oc start-build ${context.componentId} --from-dir docker --follow -n ${context.targetProject}", label : "start openshift build")
     }
+
+    def ocpbuild = sh(returnStdout: true, script:"oc get build --sort-by=.status.startTimestamp --no-headers -n ${context.targetProject} -l buildconfig=${context.componentId} | tail -n1", label : "find last build").trim().split(/\s+/)
+    
+    def ocpbuildId = ocpbuild[0]
+    def ocpbuildStatus = ocpbuild[3]
+    
+    if (ocpbuildStatus.toString().trim().toLowerCase() != "complete") {
```

Comment:
I think `oc start-build` returns with exit code 1 if it fails, so not sure if this case can happen?

Reply:
I never believe this cli .. better check 🗡